x-pack/plugins/security_solution/public/timelines/components/side_panel/event_details/index.test.tsx

```diff
@@ -169,4 +171,28 @@ describe('event details footer component', () => {
     const element = wrapper.queryByTestId('side-panel-flyout-footer');
     expect(element).toBeNull();
   });
+
+  describe('Alerts', () => {
+    const propsWithAlertIndex = {
+      ...defaultProps,
+      expandedEvent: {
+        eventId: ecsData._id,
+        indexName: `.internal.${DEFAULT_ALERTS_INDEX}-testSpace`,
+      },
+    };
+    test('it uses the alias alerts index', () => {
+      render(
+        <TestProviders>
+          <EventDetailsPanel {...{ ...propsWithAlertIndex }} />
+        </TestProviders>
+      );
+      expect(useTimelineEventsDetails).toHaveBeenCalledWith({
+        entityType: 'events',
+        indexName: `${DEFAULT_ALERTS_INDEX}-testSpace`,
+        eventId: propsWithAlertIndex.expandedEvent.eventId ?? '',
```

Comment:
nit: `?? ''` does not appear to be required:

```suggestion
        eventId: propsWithAlertIndex.expandedEvent.eventId,
```